wazuh_managers/wazuh-master-conf.yaml

```diff
@@ -328,20 +328,3 @@ data:
         <disabled>no</disabled>
       </cluster>
     </ossec_config>
-
-    <ossec_config>
```

Comment:
I'm a noob with the Ossec configuration. What are those for? :)

Reply:
Hello @JPLachance,

Wazuh has just released the trials for Wazuh SaaS and these changes are necessary for the default configuration of these environments. 

In the trials environments, we only have one Wazuh manager (of the Wazuh cluster) so we only have to make the changes in the configuration of one of them. 

Thank you for your interest and for your review. Do not hesitate to continue consulting any doubt, your contributions are enormously valuable. 

Best regards,

Alfonso Ruiz-Bravo